# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/CamerasParent.cpp
# Commit: c07caf074089
# Full Hash: c07caf07408914b5514f89c3d46fb1dced9fef06
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-13 21:39:47
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Pass TrackingId instead of nsCString to CaptureStage. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165824
# ==============================================================================

diff -r 731dcbc83284 -r c07caf074089 dom/media/systemservices/CamerasParent.cpp
--- a/dom/media/systemservices/CamerasParent.cpp	Fri Jan 13 14:25:00 2023 +0000
+++ b/dom/media/systemservices/CamerasParent.cpp	Fri Jan 13 14:25:00 2023 +0000
@@ -939,7 +939,7 @@
             }
 
             cap.VideoCapture()->SetTrackingId(
-                (*cbh)->mTrackingId.ToString().get());
+                (*cbh)->mTrackingId.mUniqueInProcId);
             error = cap.VideoCapture()->StartCapture(capability);
 
             if (!error) {